src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@using EA.Weee.Web.Areas.Aatf.ViewModels
+@using EA.Weee.Web.Extensions
+
+@model EA.Weee.Web.Areas.Scheme.ViewModels.ManageEvidenceNotes.ViewAndTransferEvidenceViewModel
+
+
+<section id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            @Html.Partial("_ManageEvidenceNotesHeaderAndTabs", Model)
+
+            <section class="govuk-tabs__panel" id="pcsDetails">
+                <h2 class="govuk-heading-m">
+                    View all reviewed evidence or transfer to another PCS
+                </h2>
+
+                <table id="edit-draft-returned-notes-table" class="govuk-table">
+                    <caption class="govuk-table__caption govuk-visually-hidden">View all reviewed evidence or transfer to another PCS</caption>
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header">
+                                Reference ID
+                            </th>
+                            <th class="govuk-table__header">
+                                Submitted date
+                            </th>
+                            <th class="govuk-table__header">
+                                Submitted by
+                            </th>
+                            <th class="govuk-table__header">
+                                Type of waste
+                            </th>
+                            <th class="govuk-table__header">
+                                Status
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody class="govuk-table__body">
+                        @if (Model.EvidenceNotesDataList.Any())
+                        {
+                            foreach (EditDraftReturnedNote note in Model.EvidenceNotesDataList)
+                            {
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell">@note.ReferenceDisplay</td>
+                            <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : "-")</td>
+                            <td class="govuk-table__cell">@note.SubmittedBy</td>
+                            @if (note.TypeOfWaste.HasValue)
+                            {
+                                <td class="govuk-table__cell">@(note.TypeOfWaste.Value.ToDisplayString()) </td>
+                            }
+                            else
+                            {
+                                <td class="govuk-table__cell"> </td>
+                            }
+                            <td class="govuk-table__cell">@note.SubmittedBy</td>
```

Comment:
is that correct? is not meant to be a Status?

Reply:
> is that correct? is not meant to be a Status?

Yeah looks like it should be

Reply:
Updated